robojumperSquadSelect/Src/robojumperSquadSelect/Classes/robojumper_UISquadSelect_SoldierPanel.uc

```diff
@@ -259,6 +259,10 @@ static function string GetRankIcon(int iRank, name ClassName, XComGameState_Unit
 {
 	if (Unit.IsSoldier())
 	{
+		if (class'robojumper_SquadSelectConfig'.static.IsCHHLMinVersionInstalled(1, 5))

```

Comment:
```suggestion
		if (class'robojumper_SquadSelectConfig'.static.IsCHHLMinVersionInstalled(1, 18))

```
That hook was introduced later. Doesn't really matter because anything below 1.15 literally won't even run but might as well use the correct value here.